util/fipstools/delocate/README.md

```diff
@@ -0,0 +1,22 @@
+### Convert delocate.peg to delocate.peg.go
```

Comment:
Just curious, was the process described in README done before? Also is it specific to ARM?

Reply:
Could you please share a link to the BSSL history that mentions this?

Reply:
https://github.com/google/boringssl/blame/302ef5ee124a123a18b8a2fd9a6b6167f4a0e65a/util/fipstools/delocate.peg.go#L3

If more history is needed, you can track the history tab of delocate related files (`delocate.go` and so on).

Bssl uses [`go:generate` command, which is released in Go 1.4](https://blog.golang.org/generate). I use `go build` to avoid the Golang update. The build is also mentioned in README.md of `peg` GitHub repo.

Reply:
thanks, I'll take a look